ultralytics/data/augment.py

```diff
@@ -1434,3 +1435,77 @@ def __call__(self, im):
         im = im.half() if self.half else im.float()  # uint8 to fp16/32
         im /= 255.0  # 0-255 to 0.0-1.0
         return im
+
+
+class RandomRotation90:
+    """
+    Applies a 90 degree rotation to the image and labels (bboxes, segments, keypoints).
+
+    Clockwise or counter-clockwise randomly.
+    """
+
+    def __init__(self, p: float = 0.0):
+        """
+        Initializes the RandomRotation90 class with probability.
+
+        Args:
+            p (float, optional): The probability of applying the rotation.
+            Must be between 0 and 1, default is 0.0.
```

Comment:
Perhaps the docstring here is not [properly formatted](https://docs.ultralytics.com/help/contributing/#google-style-docstrings)? I noticed this line from the CI

```
WARNING -  griffe: ultralytics/data/augment.py:1452: Failed to get 'name: description' pair from 'Must be between 0 and 1, default is 0.0.'
```

Reply:
that helped, thanks!